src/index.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { Provider } from 'react-redux';
-import { MuiThemeWrapper } from './components/MuiThemeWrapper';
-import { store } from './app/store';
+import { MuiThemeWrapper } from './components/MuiThemeWrapper/MuiThemeWrapper';
```

Comment:
why don't you create an index.js file to avoid name duplications in the import statement pass?